docs/options-in-versions

```diff
@@ -119,6 +119,7 @@
 --max-redirs                         7.5
 --max-time (-m)                      4.0
 --metalink                           7.27.0
+--mime-escape                        7.80.0
```

Comment:
It could make sense to have it called `--form-` something since the other formpost options start with that. Possibly quite simply `--form-backslash` ?

Reply:
--form-backslash is not very clear about dealing with escaping. But if you prefer, I can set that.

Reply:
Is noe `--form-escape`.